apps/mobile/src/features/ReviewTx/common/hooks/useOnConfirm.tsx

```diff
@@ -86,7 +86,7 @@ export const useOnConfirm = ({
       }
 
       openModal({
-        title: strings.signTransaction,
+        title: strings.swap.signTransaction,
```

Comment:
if sign transaction is repeating across all features, create a key in global.signTransaction